com.zen.browser.flatpak.yml

```diff
@@ -0,0 +1,66 @@
+app-id: com.zen.browser
```

Comment:
zen.com seems to be unrelated to this browser? https://www.zen.com/ all I see is currency exchange.

Do you control the website? How do I verify?